index.html

```diff
@@ -41,6 +41,111 @@ <h3 class="connect"><a href="#">LET'S CONNECT</a></h3>
+					</p>
+					<ul class="programnming-languages">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<h2><a href="#">See Project</a></h2>
+				</li>
+			</ul>
+			<ul class="big-project">
+				<li>
+					<img src="Assets/images/Portfolio4.png" alt="multipost-stories">
+					<h2>Multi-Post<span>Stories</span></h2>
+					<ul>
+						<li>CANOPY</li>
+						<li class="skill">Back End Dev</li>
+						<li class="year">2015</li>
+					</ul>
+					<p>
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<ul class="programnming-languages">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<h2><a href="#">See Project</a></h2>
+				</li>
+			</ul>
+			<ul class="work">
+				<li>
+					<img src="Assets/images/Portfolio1.png" alt="Gymnastic">
+					<h2>Tonic</h2>
+					<ul>
+						<li>CANOPY</li>
+						<li class="skill">Back-end dev</li>
+						<li class="year">2015</li>
+					</ul>
+					<p>
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<ul class="programnming-languages">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<h2><a href="#">See Project</a></h2>
+				</li>
+			</ul>
+			<ul class="big-project">
+				<li>
+					<img src="Assets/images/Portfolio3.png" alt="Professional-art">
+					<h2>Multi-Post<span>Stories</span></h2>
+					<ul>
+						<li>CANOPY</li>
+						<li class="skill">Back End dev</li>
+						<li class="year">2015</li>
+					</ul>
+					<p>
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<ul class="programnming-languages">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<h2><a href="#">See Project</a></h2>
+				</li>
+			</ul>
+		</section>
+		<section class="contact">
+			<h1>About<span>Myself</span></h1>
+			<p>Hello I’m a software developer! I can help you build a product, feature or website Look through some of
+				my
+				work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+				contact
+				me.
+			</p>
+			<section class="my-contact">
+				<h3 class="connect"><a href="#">LET'S CONNECT</a></h3>
+			</section>
+			<div class="link-container">
+				<ul>
+					<li><a href="#"><img src="Assets/Icons/Vector.png" alt="twitter-icon"></a></li>
+					<li><a href="#"><img src="Assets/Icons/Linkedin_icon.png" alt="Linkedin"></a></li>
+					<li><a href="#"><img src="Assets/Icons/Vector(2).png" alt="Medium"></a></li>
+					<li><a href="#"><img src="Assets/Icons/Vector(1).png" alt="GitHub"></a></li>
+					<li><a href="#"><img src="Assets/Icons/Vector(3).png" alt="AngelList"></a></li>
+				</ul>
+			</div>
+			<h2><a href="#">Get my resume</a></h2>
+			<img src="Assets/images/SkillList.png" alt="skills-pics">
```

Comment:
- [ ] You put the whole technology category in a single screenshot image. The shortcut didn't work. :laughing: :+1: Please work on this section with the appropriate HTML elements.